spider/scrapy-tsest.py

```diff
@@ -0,0 +1,20 @@
+from apscheduler.schedulers.blocking import BlockingScheduler
+from datetime import datetime
+import os
+import time
+
+def run_spider():
+    print(datetime.now().strftime("%H:%M:%S")," Run the scrapy")
+    os.system("scrapy crawl quotes")
+    time.sleep(5)
+
+
+if __name__=="__main__":
```

Comment:
Normally unit tests are written under test frameworks so they can run in batch, you can see summarized results and you have better tools to verify the results are expected.